antarest/study/web/study_data_blueprint.py

```diff
@@ -198,6 +197,24 @@ def create_link(
         params = RequestParameters(user=current_user)
         return study_service.create_link(uuid, link_creation_info, params)
 
+    @bp.put(
+        "/studies/{uuid}/links",
```

Comment:
I think we should use `link` and not `links` as it's only used for one link

Reply:
We should be consistent for all link-related endpoints (can be seen as the "links API"): better to keep `links` here.
However, I think it would make more sense to have either:
- a URL for updating only one link with the link ID in its path:
```studies/{uuid}/links/{link_id}```
- or this URL for updating a list of links